wrapper/package.json

```diff
@@ -1,29 +0,0 @@
-{
-  "name": "terraform.js",
-  "version": "0.0.1",
-  "description": "JavaScript wrapper for terraform binary",
-  "license": "MPL-2.0",
-  "publisher": "hashicorp",
-  "main": "terraform.js",
-  "scripts": {
-    "test": "semistandard --env jest && jest --coverage",
-    "lint": "semistandard --env jest --fix",
-    "build": "ncc build terraform.js --out dist"
-  },
-  "author": "",
-  "dependencies": {
-    "@actions/core": "^1.10.0",
-    "@actions/exec": "^1.1.0",
```

Comment:
Does this need to be added to the main `package.json` now?

Reply:
Ah looks like it was working because it was transitively being installed:

```
❯ npm list '@actions/exec'
setup-terraform@2.0.2 /Users/bflad/src/github.com/hashicorp/setup-terraform
└─┬ @actions/tool-cache@2.0.1
  └── @actions/exec@1.1.1
```

Since its an explicit require in `wrapper/terraform.js`, will add to `package.json` for direct management.